packages/ra-ui-materialui/src/list/Placeholder.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from 'react';
```

Comment:
you should export this component in index.js

Reply:
Done